### PR TITLE
Create 2024-09-24.md

### PR DESCRIPTION
**Pull request by RachL**

### CODE REVIEW

File: meetings/2024/2024-09-24.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- @lecoqlibre
+- @RachL
+
+## Connector codegen PR's and Data capture 
+
+Two PR need review and were added to the board by Garethe:
+
+https://github.com/datafoodconsortium/connector-codegen/pull/20 and https://github.com/datafoodconsortium/connector-codegen/pull/23
+
+These PR are based on the old and unstable version of the connector and semantizer. A new version of the semantizer will be release on September 29th.
```
Comment (RaggedStaff) [inline meetings/2024/2024-09-24.md:27]: I think it was just the semantizer that was an issue, not the connector. @lecoqlibre can you confirm?

**Review by jgaehring — APPROVED** [review]

File: meetings/2024/2024-09-24.md (hunk 1 above)
Comment (lecoqlibre) [reply meetings/2024/2024-09-24.md:27]: Yes.

**Review by lecoqlibre — APPROVED** [review]

**Review by jgaehring — APPROVED** [review]

**Comment by jgaehring** [comment]
It occurred to me yesterday while @RaggedStaff and I were speaking, would it make things easier to merge these by doing one or both of the following?

1. Move these notes into the `datafoodconsortium/.github` repository; that way it will only require one review before merging, rather than the 2 reviews required by `standard`.
2. Create a persistent branch, such as `projects/tech-meetings`, which could be linked to in the [Tech Meetings Project Board](https://github.com/orgs/datafoodconsortium/projects/5/views/2), rather than pointing to the `main` or `master` branch; that way branch protections can be fine-tuned or entirely removed regardless of which repository the notes are in. I'm using the example `projects/tech-meetings` here with [slash branches](https://www.baeldung.com/ops/git-refs-branch-slash-name) as a prefix category (potentially w/ branches `projects/main` and `projects/prototype` added now or later), but that's not necessary.

I've prepped a change to the `.github` repo I can submit as a PR, with all the latest changes including the commits @lecoqlibre added yesterday, so just say the word and I can get that ball rolling.

**Comment by RachL** [comment]
Sounds good to me! 💪

**Review by RaggedStaff — APPROVED** [review]
LGTM!

**Pull request merged**